service/history/workflow/mutable_state_rebuilder_test.go

```diff
@@ -1778,7 +1811,7 @@ func (s *stateBuilderSuite) TestApplyEvents_EventTypeExternalWorkflowExecutionSi
 
 func (s *stateBuilderSuite) TestApplyEventsNewEventsNotHandled() {
 	eventTypes := enumspb.EventType_value
-	s.Equal(41, len(eventTypes), "If you see this error, you are adding new event type. "+
+	s.Equal(42, len(eventTypes), "If you see this error, you are adding new event type. "+
```

Comment:
Can you change `stateBuilderImpl` to `MutableStateRebuilderImpl` in error message below.